roles/chronos/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# right now there is an issue with chronos zookeeper connection with auth, so leave it blank for now
```

Comment:
can you be more specific here? What's the issue? If there's an open ticket, could you link to it please?